### PR TITLE
myeong's intro

### PR DESCRIPTION
**Pull request by myeong**
This is the first draft of my introduction

### CODE REVIEW

File: _posts/2016-06-24-myeong-intro.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+title:  "Myeong: introduction"
```
Comment (arokem) [inline _posts/2016-06-24-myeong-intro.md]: For consistency, please change this to "Introducing DSSG Fellow Myeong Lee"

File: _posts/2016-06-24-myeong-intro.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+date:   2016-06-24
+author: Myeong Lee
+---
+<link href="../assets/css/myeong.css" rel="stylesheet"></link>
+
+# Myeong Lee (Ph.D. Student)
```
Comment (arokem) [inline _posts/2016-06-24-myeong-intro.md]: You don't need a title. That gets automagically generated from the `title` field in the header!

File: _posts/2016-06-24-myeong-intro.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+title:  "Myeong: introduction"
+date:   2016-06-24
+author: Myeong Lee
+---
+<link href="../assets/css/myeong.css" rel="stylesheet"></link>
```
Comment (arokem) [inline _posts/2016-06-24-myeong-intro.md]: CSS, huh? You win. 

(does it actually work?)

File: _posts/2016-06-24-myeong-intro.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...

I hope to learn how to deal with big urban data, how to connect conceptual variables to quantifiable measures, and how to build a robust and generalizable model for prediction. So far, I have learned how to speed up the computation using Python's pandas library, how to store and visualize geospatial data using PostgresSQL database and QGIS, and how to connect them each other. This series of lectures and trainings were very helpful for filling gaps in my knowledge about geospatial databases/visualizations.
```
Comment (arokem) [inline _posts/2016-06-24-myeong-intro.md]: Maybe you could tell us a bit about why you're at the DSSG and what you hope to accomplish and/or learn over the course of the summer? If you want, you can tell us what you've been doing so far, and maybe what you've already learned?

**Comment by arokem** [comment]
Spoke too soon - I see that you've added content!

File: _posts/2016-06-24-myeong-intro.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+UW's DSSG this summer attracted me a lot, because the topics and skillsets that the program covered were perfectly fit to my interests and research directions. Especially, the project that I am working on, CrowdCensus, is an urban/social computing project that makes use of various kinds of geo-tagged data such as cellphone detail records (CDR) and social media data to predict socio-economic status of urban regions. Through this project, I am willing to build an easy-to-use model and system that can help policy-makers and urban planners, but also understand the data themselves so to better conceptualize geo-tagged data with a theoretical framework. I am really glad that I can work with brilliant people from diverse academic disciplines.
```
Comment (arokem) [inline _posts/2016-06-24-myeong-intro.md]: The word "willing" might not be the best word here. How about "hoping"? Or "working"?

File: _posts/2016-06-24-myeong-intro.md (hunk 3 above)
Comment (myeong) [inline _posts/2016-06-24-myeong-intro.md]: actually it didn't work, so I removed it

Comment (myeong) [inline _posts/2016-06-24-myeong-intro.md]: it works fine in MacDown, but I think Github page does not read a css file from MD file.. somehow..

**Pull request merged**